WordPress/Classes/Networking/DomainsRemote.swift

```diff
@@ -0,0 +1,155 @@
+import Foundation
+
+class DomainsRemote: ServiceRemoteREST {
```

Comment:
I believe we're still following the pattern of `[Entity]ServiceRemote` so you might want to refactor this class name.